chainer/functions/math/sparse_matmul.py

```diff
@@ -0,0 +1,486 @@
+import chainer
+from chainer.backends import cuda
+from chainer import function_node
+from chainer import utils
+from chainer.utils import type_check
+import numpy
+import warnings
```

Comment:
Could you reorder the imports? Here's an example https://github.com/chainer/chainer/blob/master/chainer/utils/array.py ?

Reply:
will fix in that way!